IFC4x3/Sections/Resource definition data schemas/Schemas/IfcCostResource/Entities/IfcAppliedValue/Documentation.md

```diff
@@ -12,3 +12,8 @@ Applied values may be referenced from a document (such as a price list). The rel
 
 { .change-ifc2x4}
 > IFC4 CHANGE&nbsp; Entity made non-abstract to support general formula expressions of constraints, data type of date-based attributes changed into _IfcDate_, _ValueType_ and _Condition_ promoted from _IfcCostValue_, _Components_ and _ArithmeticOperator_ attributes added to replace _IfcAppliedValueRelationship_ for more efficient encoding and reference tracking.
+
+{ .spec-head}
+Informal propositions
+
+1. When the *ArithmeticOperator* equals `MODULO``, the list of *Components* shall contain exactly two values, which shall be positive integers.
```

Comment:
```suggestion
1. When the *ArithmeticOperator* equals `MODULO`, the list of *Components* shall contain exactly two values, which shall be positive integers.
```

Reply:
This is "mildly" inappropriate. Your linked issue in which you reference the change has exactly this wording (which I copied over). Please be kind enough to fix your issue first.